### ISSUE TITLE
Accidentally declined microphone permissions, now cannot record anything using the desktop app.

### ISSUE THREAD

**Issue by JustABill1** [braden-w/whispering#68]
I was holding "Space" when I tried to use the application, and it accidentally declined microphone permissions. Whispering isn't listed as an application under the Microphone Privacy Settings, so I cannot restore permissions. I tried to find a fix somewhere in the Registry Editor, but no luck. Uninstalling and reinstalling also did not help. 

Is there some way to have the application check for microphone permissions and re-request access?

**Comment by CaiCharley** [comment]
I'm having the same issue. Did you find a solution?

**Comment by CaiCharley** [comment]
It seems if you delete the /appdata/local/com.bradenwong.whispering folder and reinstall it allows the permission to pop up again. However, even when I clicked allow, the microphone still doesn't turn into the red square for me and doesn't allow it to work...

**Comment by JustABill1** [comment]
No, I couldn't find a solution, sadly. I ended up relying on the webpage, which was recently shut down so now I am using the Google Chrome extension. Not as efficient, but hopefully will be fixed in the future.

**Comment by JustABill1** [comment]
I tried your solution, and I actually found that it worked. After confirming deletion of those files, I reinstalled whispering using the .msi file. After putting in my API key and then clicking the microphone, the permission popped up, I hit allow, and the transcription actually went through.

I don't know what would be working different for you. I noticed when I went to the uninstaller that there were two instances of 'Whispering'. I uninstalled both and their uninstallers seemed slightly different (not sure why).
